### PR TITLE
Companion fix for apigee/apigee-client-php/pull/334

### PR DESCRIPTION
**Pull request by mxr576**
Add missing try-catch block around GCE authentication availability check.

Follow up tasks:
- [ ] Bump minimum required PHP API client version after https://github.com/apigee/apigee-client-php/pull/334 was merged and released in a tagged version

### CODE REVIEW

**Review by shishir-intelli — APPROVED** [review]
LGTM

File: src/Plugin/KeyInput/ApigeeAuthKeyInput.php [hunk 1]
```diff
@@ -133,7 +134,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
 
     $state_for_not_gcp_hosted = [];
     $gceServiceAccountAuth = new GceServiceAccountAuthentication(\Drupal::service('apigee_edge.authentication.oauth_token_storage'));
-    if ($gceServiceAccountAuth->isAvailable()) {
+    $is_gce_hosted_site = FALSE;
+    try {
+      $is_gce_hosted_site = $gceServiceAccountAuth->isAvailable();
+    } catch (Exception) {
```
Comment (shishir-intelli) [inline src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: Newline after closing curly brace.

Comment (shishir-intelli) [reply src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: image

Comment (mxr576) [reply src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: Thx for reporting but why this violation is not spotted by GHA? :)

Comment (shishir-intelli) [reply src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: In our forked repository it works, but in the 3.x branch the GHA is not checking the PR files,
(we will work on it.)
See https://github.com/shishir-intelli/apigee-edge-drupal/actions/runs/7408909761/job/20158083977

Meanwhile, please setup GHA in your repository as well.

Comment (mxr576) [reply src/Plugin/KeyInput/ApigeeAuthKeyInput.php]: > Meanwhile, please setup GHA in your repository as well.

You are right, I need to double check if that is working for not, but for quick hotfixes like this, I just use Github UI and hope that any and every issue on my change will be reported by GHA running on this repo/branch.

**Pull request merged**